### DISCUSSION TITLE
Krita

### DISCUSSION THREAD

**Discussion by AnubisNekhet** [catppuccin/catppuccin#1926] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the requested port be themed?

https://github.com/IBArbitrary/krita-themes/blob/master/KritaDraculaDark.colors
This is a good starting point, it should be fairly easy.

### Has the requested port been themed already?

Nope

**Comment by Sourcastic** [comment]
Yello,
As far as I can tell, this is just a *.colors file that the KDE Plasma Desktop uses to theme its applications. I've attached the flavours you can use and install using similar instructions that are in the readme of the repo you linked. You'll have to convert the file extension to .colors from .txt because GitHub doesn't allow me to attach .color files.

[CatppuccinFrappeFlamingo.txt](https://github.com/catppuccin/catppuccin/files/10735473/CatppuccinFrappeFlamingo.txt)
[CatppuccinLatteFlamingo.txt](https://github.com/catppuccin/catppuccin/files/10735474/CatppuccinLatteFlamingo.txt)
[CatppuccinMacchiatoFlamingo.txt](https://github.com/catppuccin/catppuccin/files/10735475/CatppuccinMacchiatoFlamingo.txt)
[CatppuccinMochaFlamingo.txt](https://github.com/catppuccin/catppuccin/files/10735476/CatppuccinMochaFlamingo.txt)

On another note: Problem is, the KDE Port generates the flavours and accents using an install script. I didn't consider using them for Krita like this. Probably will add them to releases moving forward and add instructions in the readme, I suppose?

**Comment by Sourcastic** [comment]
The KDE Port now includes instructions for Krita. I don't know how search-able this is if you're coming from the main repo, but it's _a_ solution.
@sgoudham do advise. If you feel this was good enough, I think we can close the discussion.

**Reply by sgoudham** [reply]
Thanks for taking the lead on this! Will close now
